iMichaelOwolabi/my-diarium-backend
# 168093846 integrate database

#### What does this PR do?
Integrate postgres database
#### Description of Task to be completed?
* Integrate Postgres DB
* Configure Sequelize ORM
#### How should this be manually tested?
Clone the project repo and cd into the directory
- run npm  install to install all dependencies
- run npm test
- you should see the database file being migrated
#### Any background context you want to provide?
This task is just to setup the database for the project
#### What are the relevant pivotal tracker stories?
[#168093846](https://www.pivotaltracker.com/story/show/168093846)
